web/yarn.lock

```diff
@@ -2776,7 +2776,7 @@ commander@2.17.x:
   resolved "https://registry.yarnpkg.com/commander/-/commander-2.17.1.tgz#bd77ab7de6de94205ceacc72f1716d29f20a77bf"
```

Comment:
pls delete this file, which should be generated automatically. it is not a source file.

Reply:
yes, I've just removed